build.gradle

```diff
@@ -613,6 +622,43 @@ Bundle-Version: $version
     }
 }
 
+project (':quasar-kotlin') {
+
+    apply plugin: 'kotlin'
+
+    ext.moduleName = 'co.paralleluniverse.quasar.kotlin'
+
+    configurations.all {
+        resolutionStrategy {
+            dependencySubstitution {
+                substitute module('org.jetbrains.kotlin:kotlin-stdlib') with module("org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion")
+                substitute module('org.jetbrains.kotlin:kotlin-stdlib-common') with module("org.jetbrains.kotlin:kotlin-stdlib-common:$kotlinVersion")
+            }
+        }
+    }
+
+    dependencies {
+        implementation project(path: ':quasar-core', configuration: 'shadowedJar')
+        implementation project(':quasar-actors')
+
+        implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
+        implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion"
```

Comment:
Can we please not put the `quasar-kotlin` project's `build.gradle` file in here? It's not needed for this PR, and just makes the build even more atypical.